jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -310,6 +310,9 @@
           status-add-test-results: false
           trigger-phrase: 'verify|test'
           cmd: |
+            # verify-godeps compares against upstream, but remote/master might be stale
+            # if .git was retained between runs. Update it explicitly here.
+            git fetch remote master:refs/remotes/remote/master
```

Comment:
once again I demonstrate that I don't understand git...

should we just be changing the `scm` section of the YAML to make sure it fetches master?